web/src/games/bullsAndCows/locales/en.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "Bulls & Cows",
+  "description": "similar to Mastermind",
+  "descriptionTag": "Play Bulls and Cows (also known as Mastermind) for free online.",
```

Comment:
The wording should be "similar to Mastermind", not also known as.

Reply:
Done